src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -183,10 +183,12 @@ public override bool Equals(object obj)
         /// Converts the value of this instance to a <see cref="string"/>.
         /// </summary>
         /// <remarks>
-        /// Returns the underlying UTF-16 encoded string.
-        /// </remarks>
-        /// <remarks>
-        /// Returns an empty string on a default instance of <see cref="JsonEncodedText"/>.
+        ///   <para>
+        ///     Returns the underlying UTF-16 encoded string.
+        ///   </para>
+        ///   <para>
+        ///     Returns an empty string on a default instance of <see cref="JsonEncodedText"/>.
+        ///   </para>
```

Comment:
Can we simplify this to just be something like:
```C#
Returns the underlying string, or the empty string if there is none.
```
?

Reply:
Am I allowed to change the contents? I merely merged all the two separate remarks into one remark  but with two paragraphs. It's a fair opinion though, IMO.

Reply:
Ended up removing the first paragraph entirely; to me, it seemed unnecessary because the summary bit already implies about it.